src/components/notes-listing/note-card/NoteCard.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { useState, useRef, useEffect } from "react";
 import { useNotes } from "../../../context/NotesContext";
```

Comment:
you can do a barrel import / export , makes the code look cleaner

Reply:
yes.